src/test/modules/dummy_index_am/meson.build

```diff
@@ -12,15 +10,16 @@ endif
 
 dummy_index_am = shared_module('dummy_index_am',
   dummy_index_am_sources,
-  kwargs: pg_mod_args,
+  kwargs: pg_mod_args_no_install,
 )
 testprep_targets += dummy_index_am
+test_install_libs += dummy_index_am
 
-install_data(
+dummy_index_am_install_sources = files(
   'dummy_index_am.control',
   'dummy_index_am--1.0.sql',
-  kwargs: contrib_data_args,
 )
+test_install_files += dummy_index_am_install_sources
 
```

Comment:
We don't need dummy_index_am_install_sources as a variable, right? So it seems like it'll be nicer to write this as
```meson
test_install_files += files(
  'dummy_index_am.control',
  'dummy_index_am--1.0.sql',
)
```

Reply:
Yes, we don't need these right now but I did these to make it easier for possible future needs like windows icon task.

Reply:
I think the cost in verbosity is higher than the chance of needing those variables.